__tests__/app.test.js

```diff
@@ -10,25 +10,15 @@ beforeEach(() => {
 
 afterAll(() => db.end())
 
-describe('GET /api/topics', () => {
-	test('GET - status 200 - returns an array containing the right amount of data', () => {
-		return request(app)
-			.get('/api/topics')
-			.expect(200)
-			.then(({ body }) => {
-				expect(body.topics.length).toBe(3)
-			})
-	})
 
-	test('GET - status 200 - returns an array containing the right type of data (slug; description)', () => {
+describe('GET /api', () => {
+	test('returns a JSON file with all the information provided by a JSON file', () => {
 		return request(app)
-			.get('/api/topics')
+			.get('/api')
 			.expect(200)
 			.then(({ body }) => {
-				body.topics.forEach((topic) => {
-					expect(typeof topic.slug).toBe('string')
-					expect(typeof topic.description).toBe('string')
-				})
+				expect(typeof body).toBe('object')
```

Comment:
You could also require in the JSON file and test if it matches the one that's being served.